src/lib/wallet.js

```diff
@@ -55,6 +55,7 @@ async function initMiddleware() {
     },
   };
   spec.basePath = '/mdw//';
+  spec.schemes = ['https'];
```

Comment:
This issue resolved in https://github.com/aeternity/aepp-sdk-js/commit/3a876bbb80300cbf91891daa642f6daee61c1427, but it's included in 0.9.0 release which is in draft. Propose this workaround before release.

Reply:
This fixes lack of loading of transactions and names (which we get from middleware), because recent changes in swagger in sdk introduces custom specification https://github.com/aeternity/aepp-sdk-js/commit/683082b3330c9da17dd1b3b05a34d3f174ffa27b#diff-95fa4998e94b57e3debb9d34aaf93a764251db8e76e5e067a7c1315aa025a3ab but our specification doesn't include https sceme + it doesn't pass specific url check and therefore no scheme is set for our spec.

Reply:
And because of that we were making requests to `http://` middleware which doesn't pass CORS and we weren't getting any info from middleware